src/opera/pge/dswx_s1/templates/dswx_s1_measured_parameters.yaml

```diff
@@ -0,0 +1,134 @@
+AREA_OR_POINT:
+  description: Indicates that pixel values are assumed to represent an area rather than points.
+CONTACT_INFORMATION:
+  description: OPERA Project contact email address.
+DSWX_PRODUCT_VERSION:
+  description: The DSWx-S1 product version.
+INSTITUTION:
+  description: Name of the generating Institution.
+LAYOVER_SHADOW_COVERAGE:
+  description: The percentage of layover and shadow in the DSWx-S1 product based on OPERA RTC-S1 product.
+MGRS_COLLECTION_ACTUAL_NUMBER_OF_BURSTS:
+  description: The number of actual RTC-S1 bursts that are collected within the MGRS tile collection ID.
+MGRS_COLLECTION_EXPECTED_NUMBER_OF_BURSTS:
+  description: The number of expected RTC-S1 bursts that are supposed to be collected within the MGRS tile collection ID.
+MGRS_COLLECTION_MISSING_NUMBER_OF_BURSTS:
+  description: The number of missing RTC-S1 bursts.
+MGRS_POL_MODE:
+  description: "The description for the polarizations of the collected bursts. 'MIX_DUAL_POL': ['HH', 'HV', 'VV', 'VH'], 'MIX_DUAL_H_SINGLE_V_POL': ['HH', 'HV', 'VV'], 'MIX_DUAL_V_SINGLE_H_POL': ['VV', 'VH', 'HH'], 'MIX_SINGLE_POL': ['HH', 'VV'], 'DV_POL': ['VV', 'VH'], 'SV_POL': ['VV'], 'DH_POL': ['HH', 'HV'], 'SH_POL': ['HH'],"
+INPUT_DEM_SOURCE:
+  description: Description of the input DEM
```

Comment:
A nitpick, but I can see someone complaining about this in the future: lets be consistent on period usage for these descriptions and always include one even if it was left out of the text copied from the product spec

Reply:
Done